lib/main.coffee

```diff
@@ -55,12 +55,12 @@ module.exports =
   # or the users globally installed sass-lint version
   findExecutable: ->
     {spawnSync} = require 'child_process'
-    {getPath} = require 'consistent-path'
+    consistentEnv = require 'consistent-env'
     if not @globalSassLint
       return require path.join(__dirname, '..', 'node_modules', 'sass-lint')
     if @globalPath is '' and prefixPath is null
       npmCommand = if process.platform is 'win32' then 'npm.cmd' else 'npm'
-      env = Object.assign({}, process.env, {PATH: getPath()})
+      env = Object.assign({}, process.env, consistentEnv())
```

Comment:
And I was copying it from somewhere else :stuck_out_tongue:.

I'm not even sure where now...